### PR TITLE
#906 Add SQLite3 support

### PR DESCRIPTION
**Pull request by adaur**
Thanks to @Koos10

### CODE REVIEW

**Comment by franzliedke** [comment]
Looks good, just one question: Why did you use SQLite3 instead of PDO (as I did in my branch)?

**Comment by adaur** [comment]
Oh my, I didn't notice you used PDO. I tested this locally and it works fine, I thought your solution was uncomplete.

**Comment by franzliedke** [comment]
I'm not even sure why my branch hasn't been merged yet, to be honest. ;)

Probably because I was waiting for other people to test it, too.

The only arguments I'd have in favor of PDO are these two:
* It can be used as reference for other possible drivers (that really should use PDO).
* It seems to be a little more widely distributed, judging by the installation instructions, which suggest that (at least on Windows) Sqlite3 is disabled by default.

File: include/dblayer/sqlite3.php [hunk 1]
```diff
@@ -0,0 +1,642 @@
...
+		if (!$this->link_id)
+			error('Unable to open database \''.$db_name.'\'.', __FILE__, __LINE__);
+		else
+			return $this->link_id;
+	}
+	
+	
+	function DBLayer($db_host, $db_username, $db_password, $db_name, $db_prefix, $p_connect)
+	{
+		return $this->__construct($db_host, $db_username, $db_password, $db_name, $db_prefix, $p_connect);
```
Comment (franzliedke) [inline include/dblayer/sqlite3.php]: Got a `return` here, too. ;)

**Comment by adaur** [comment]
We should really use PDO.

**Pull request closed without merging**